therapy/etl/drugbank.py

```diff
@@ -121,9 +121,11 @@ def _load_therapy(self, batch, params):
             del params['other_identifiers']
 
         for label_type in ['trade_names', 'aliases']:
-            if label_type in params and (
-                    len(params[label_type]) > 20 or not params[label_type]):
-                del params[label_type]
+            if label_type in params:
+                if not params[label_type] or len(
+                        set({a.casefold(): a for a in params[
+                            label_type]}.values())) > 20:
```

Comment:
ditto above - I think (maybe?) this could be cast directly as a set via comprehension rather than making it a dict first and then getting values

```suggestion
                        {a.casefold() for a in params[label_type]} > 20:
```